views/pages/index.ejs

```diff
@@ -212,73 +212,76 @@
                     <li class="google-plus"><a href="#"><i class="fa fa-google-plus"></i></a></li>
                     <li class="linkedin"><a href="#"><i class="fa fa-linkedin"></i></a></li>
-                  </ul>
+                  </ul> -->
                 </div>
               </div>
             </div>
           </div>
           <div class="col-lg-3 col-md-6 col-xs-12">
             <div class="single-team">
-              <img src="img/team/team2.jpg" alt="">
+              <img src="img/team/t2.jpeg" alt="">
               <div class="team-details">
                 <div class="team-inner">
-                  <h4 class="team-title">John Smith</h4>
-                  <p>CEO & Co-Founder</p>
-                  <ul class="social-list">
-                    <li class="facebook"><a href="#"><i class="fa fa-facebook"></i></a></li>
-                    <li class="twitter"><a href="#"><i class="fa fa-twitter"></i></a></li>
-                    <li class="google-plus"><a href="#"><i class="fa fa-google-plus"></i></a></li>
-                    <li class="linkedin"><a href="#"><i class="fa fa-linkedin"></i></a></li>
-                  </ul>
+                  <h4 class="team-title">Donald Clark,</h4>
+                  <p>Senior Project Manager</p>
+                  <p class="team-bio">Donald has the lead on various primary activities associated with the Cell Science technology
+                    implementation: Supporting scientists in the build out Cell Science laboratory and processes for
+                    scaling efficacy, prepping in coordination with counsel the licensing strategy, supporting IP
+                    documentation, and coordinating compliance with regulatory authorities.
+                    He is a seasoned executive with over thirty years of entrepreneurial experience in founding,
+                    funding, and scaling technology enabled start-ups.</p>
                 </div>
               </div>
             </div>
           </div>
           <div class="col-lg-3 col-md-6 col-xs-12">
             <div class="single-team">
-              <img src="img/team/team3.jpg" alt="">
+              <img src="img/team/t3.jpeg" alt="">
               <div class="team-details">
                 <div class="team-inner">                  
-                  <h4 class="team-title">Emilly Williams</h4>
-                  <p>Business Manager</p>
-                  <ul class="social-list">
-                    <li class="facebook"><a href="#"><i class="fa fa-facebook"></i></a></li>
-                    <li class="twitter"><a href="#"><i class="fa fa-twitter"></i></a></li>
-                    <li class="google-plus"><a href="#"><i class="fa fa-google-plus"></i></a></li>
-                    <li class="linkedin"><a href="#"><i class="fa fa-linkedin"></i></a></li>
-                  </ul>
+                  <h4 class="team-title">Dr. Peter Whitton</h4>
+                  <p>Lead Biochemist</p>
```

Comment:
Instead of hard coding these values you should create a profile model and have a source file (YAML file) that you can consume from assets via fetch.  (consume it as if it were a YAML response from server but instead it comes from the assets directory, etc). 

Then you can have a YAML file like the following:

```
staff:
  name: Dr. peter Whitton
    portrait: t3.jpeg
    role: Lead Biochemist
    bio: <bio here>
  name: Other Person
    portrait: t2.jpeg
    role: Terminator
    bio: He'll be back
``` 

Then you can have a cleaner template (EJS) where you iterate through it and produce form a single include

Reply:
Not a bad idea, though this will remain hardcoded for now. One day we will set up a CRM connection